bluepyemodel/tasks/emodel_creation/optimisation.py

```diff
@@ -1193,7 +1193,7 @@ def output(self):
         """ """
         checkpoint_path = get_checkpoint_path(self.access_point.emodel_metadata, seed=self.seed)
 
-        fname = f"{Path(checkpoint_path).stem}.pdf"
+        fname = f"{Path(checkpoint_path).stem}__optimisation.pdf"
```

Comment:
no it should be the same, I think we forgot to change this line when we updated the figure name (https://github.com/BlueBrain/BluePyEModel/blame/6f12c01f6a570dc61a76f89efaa4acc35715183f/bluepyemodel/emodel_pipeline/plotting.py#L205)

Reply:
Ok. The optimisation figure name can be corrected now.

Reply:
?

Reply:
`f"{Path(checkpoint_path).stem}__optimisation.pdf"` 
From here, I thought that `optimisation` figures generated by the workflow Task will have  `__optimisation` in the end and that it will different for the such figures generated by `LocalAccessPoint` ? As you mention earlier that the [line](https://github.com/BlueBrain/BluePyEModel/blame/6f12c01f6a570dc61a76f89efaa4acc35715183f/bluepyemodel/emodel_pipeline/plotting.py#L205) was not changed correctly in the previous PR. 
So, the change you made is fine. :)